### PR TITLE
[docs] Standardize docs menu items

### PR DESCRIPTION
**Pull request by CollierCZ**
## The Problem/Issue/Bug:

As part of an audit of the docs, this PR focuses on the left menu items and making sure they follow a standard pattern so readers can more easily scan for what they want to find.

## How this PR Solves The Problem:

Uses verbs and a single way of referring to DDEV to make comparisons more clear.

## Automated Testing Overview:

This only changes the docs, so shouldn't require other testing

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3891"></a>

### CODE REVIEW

**Review by rfay — CHANGES_REQUESTED** [review]
Minor suggestions, at least one that you can just click the button to commit. 

I ran this with `mkdocs serve` and looked at it and it seemed like a big improvement.

I found that spellchecking wasn't being done beyond first level of docs, so you may have to rebase this to get it in after that one goes in. 

Thanks so much! What a great start to cleanup.

I worry always that we'll change a URL that's externally known, but at least here the internal links should be checked during tests.

**Comment by rfay** [comment]
BTW I just read the [goreleaser docs](https://goreleaser.com/intro/) from start to finish on my phone. So nice, such good flow. Would love to end up with nested sections like that. And they're using our same mkdocs material theme.

**Comment by rfay** [comment]
Rebased. I figured since I made the mess I should do the rebase.

File: docs/users/topics/webserver.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
...
+It's a common practice to set up HTTP to TLS redirects in the .htaccess file, which leads to issues with the DDEV proxy setup. The TLS endpoint of a DDEV project is always the `ddev-router` container and requests areforwarded through plain HTTP to the project's webserver. This of course results in endless redirects and can never work. Therefore, you need to change the root `.htaccess` file so the Apache webserver of the project correctly handles these requests for your local development environment with DDEV. The following snippet should work for most scenarios and not just DDEV and could replace an existing redirect:
```
[the remark this answers is not in the document]
Comment (CollierCZ) [reply docs/users/topics/webserver.md]: Thanks!

**Comment by CollierCZ** [comment]
The GoReleaser docs definitely look clean. I think something similar could be possible in time, for sure. Just needs time to do a deeper run. Was hoping to get quicker improvements this time through and come back to it.

**Comment by rfay** [comment]
Thanks!

**Pull request merged**